appinfo/app.php

```diff
@@ -1,4 +1,11 @@
 <?php
+## ugly patch for issue #9 https://github.com/owncloud/user_external/issues/9
```

Comment:
I don't think this is ugly - there is no other easy way.
You can drop this comment from my point of view

Reply:
ahahahah okok... for me is ugly, 
now refactoring :)